cosmovisor/scanner.go

```diff
@@ -27,6 +27,10 @@ type UpgradeInfo struct {
 // It returns (nil, err) if the input stream errored
 // It returns (nil, nil) if the input closed without ever matching the regexp
 func WaitForUpdate(scanner *bufio.Scanner) (*UpgradeInfo, error) {
+	// set larger buffer in case the line is too long, the default one is 64*1024
+	const maxCapacity = 512 * 1024
```

Comment:
How about making it a ENV setting?

Reply:
It could work - as long as sane defaults are provided :+1:

Reply:
update to use ENV setting with defaults provided
@alessio @anilCSE